lib/Backends/CPU/libjit/libjit.cpp

```diff
@@ -1395,11 +1390,10 @@ void libjit_max_pool_xy_grad_f(float *inG, const float *outG,
 
       for (size_t ax = 0; ax < outWdims[1]; ax++) {
         for (size_t ay = 0; ay < outWdims[2]; ay++) {
-          // For the x and y argmax's, we use a 5-dimensional
-          // tensor whose fifth dimension has size 2:
-          size_t ix = 2 * libjit_getXYZW(outWdims, n, ax, ay, z);
-          size_t maxX = inXY[ix];
-          size_t maxY = inXY[ix + 1];
+          int64_t argmaxNHW =
```

Comment:
nit: Can you add comments about what's going on here?

Reply:
I just realized that this whole thing makes no sense. We extract X and Y coordinates only to recompute flattened index (which is already stored in argmax). This can be done much simpler. :)